Forta-Bot-Challenge-4/COMP01/README.md

```diff
@@ -0,0 +1,29 @@
+# Investment tracker bot for USDC
+
+## Description
+
+The COMP01 bot is a monitoring tool designed to track individual user investments over a 24-hour period, generating alerts when the user surpasses a predefined threshold during supply events. The associated proxy contract for this bot is [`0x1B0e765F6224C21223AeA2af16c1C46E38885a40`](https://etherscan.io/address/0x1B0e765F6224C21223AeA2af16c1C46E38885a40).
+
+## Purpose
+
+The primary objective is to monitor individual user addresses for a specified time frame (24hrs) and send alerts when their investments exceed a predefined threshold. This monitoring is conducted through supply events.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- **Alert ID:** COMPOUND-THRESH-123
```

Comment:
You could make the alertId be COMP-01

Reply:
Applied this suggestion to bots 2 and 3 as well.